src/NuGet.Clients/NuGet.SolutionRestoreManager.Interop/NuGet.SolutionRestoreManager.Interop.csproj

```diff
@@ -43,13 +43,13 @@
   <Import Project="$(BuildCommonDirectory)common.targets" />
 
   <!-- Custom pack, called by build.proj -->
-  <Target Name="Pack" DependsOnTargets="Build">
+  <Target Name="Pack">
```

Comment:
@mishra14 had trouble converting it during the initial migration, but we could try again.

I used ``GeneratePackageOnBuild`` early on for all projects but then moved to explicit pack target calls. The reason is that we run four different builds VS14, VS15, VS15-insertable, and VS15-Release but only want to pack for VS15 and VS15-Release. It also caused building for tests to generate nupkgs.